roles/network-multus/tasks/deprovision.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: openshift cni config
+  set_fact:
+    cni_config: "{{ openshift_cni_config }}"
+  when: cluster=="openshift"
+
+- name: kubernetes cni config
+  set_fact:
+    cni_config: "{{ kubernetes_cni_config }}"
+  when: cluster=="kubernetes"
+
+- name: Render multus deployment yaml
+  template:
+    src: multus.yml
+    dest: /tmp/multus.yml
+
+- name: Delete multus Resources
+  command: kubectl delete -f /tmp/multus.yml --ignore-not-found
+
+- name: Render cni plugins deployment yaml
+  template:
+    src: cni-plugins.yml
+    dest: /tmp/cni-plugins.yml
+
+- name: Delete cni plugins Resources
+  command: kubectl delete -f /tmp/cni-plugins.yml --ignore-not-found
+
+- name: Render L2 plugin deployment yaml
+  template:
+    src: l2-bridge.yml
```

Comment:
we should not deploy the L2 bridge, certainly not in this PR. Only Multus.

We are sure about ovs-cni, so we do not need to deploy L2-bridge at all imo.

Reply:
Good point because we have @leongold PR #371 to deploy ovs-cni